mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
> But more objectively, I'm not really sure why that's a really useful behavior given how similar it would be to the "auto" mode that distributes evenly across time.
Good point, I give up.

> Maybe auto mode should actually behave that way, looking for peak values in each tenth of the data.
I don't think so - it should be very predictable IMO - but if the peak finder becomes a separate function, it should be simple to e.g. allow a 3rd keyword, like "representative", for a mix between evenly-spaced and peaks.